crates/symbolicator-service/src/caching/fs.rs

```diff
@@ -330,3 +377,19 @@ where
         },
     }
 }
+
+/// Returns the corresponding metadata file path
+/// for a cache file.
+pub(crate) fn metadata_path(path: impl AsRef<Path>) -> PathBuf {
+    let mut out = path.as_ref().to_path_buf();
+    out.set_extension("md");
```

Comment:
Markdown?^^

Maybe (no clue if `set_extension` allows you to include a dot):
```suggestion
    out.set_extension("metadata.json");
```